cmd/secrets.go

```diff
@@ -2,13 +2,12 @@ package cmd
 
 import (
 	"fmt"
-
 	"github.com/appuio/seiso/cfg"
 	"github.com/appuio/seiso/pkg/cleanup"
-	"github.com/appuio/seiso/pkg/openshift"
+	"github.com/appuio/seiso/pkg/kubernetes"
+	"github.com/appuio/seiso/pkg/service"
```

Comment:
I would call it the "secret" package. with the Interface name to just "Interface" so that it becomes "secret.Service" and your Implementation name becomes "secret.SecretsService"

Although Secrets and ConfigMaps have similar properties, they are not the same and should be treated separately